### ISSUE TITLE
Website Testing

### ISSUE THREAD

**Issue by seguinryan** [CajSWehst/web-design-2-independent-project#9]
Hey Caj,

Your site is looks good, good use of fonts and colours, and navigation seems to be running! Good job man, it's a whole lot better than mine! 

Issue #1: Logo
• On the upper left corner of your page, you have your logo, you should make this a directory to bring you back to your home page.

Issue #2: Website Navigational Links
• After choosing to go to a social media link, it directs you back to the top of your page, and not on Instagram or LinkedIn.  Also, on your home page you have two navigational buttons, "Our Work" and "Learn More". 
"Learn More" should be renamed as "About" page, as that is its directory, and you should change "Our Work" to "My Portfolio". Your navigations are running fine, but you should change how your site is navigating, and how it is prioritized. It's a little hard to follow.

Issue #3: Get in Contact
• At the bottom of your about page, you have a "Get in Contact" section, which should be pulled further to the top of your about page so people can easily get in touch with you without having to get distracted by your "Type of Work" section. Your sections are a little different, but if you look over your sitemap and move a few sections around, you should have a more functional and captivating and well designed website down in no time.

**Comment by thatkidmadrid** [comment]
https://learn-the-web.algonquindesign.ca/courses/shared/browser-testing/#create-a-detailed-issue
